src/results/electric_tariff.jl

```diff
@@ -14,7 +14,41 @@
 - `year_one_export_benefit_before_tax` export credits over the first year, before considering tax benefits. A positive value indicates a benefit. 
 - `lifecycle_coincident_peak_cost_after_tax` lifecycle coincident peak charge in present value, after tax
 - `year_one_coincident_peak_cost_before_tax` coincident peak charge over the first year
+
+Outputs related to electric tariff (year-one rates and costs; not escalated):
+- `monthly_fixed_cost_series_before_tax` the fixed monthly cost of electricity for modeled meter per chosen electric tariff in \$/month
+- `energy_rate_series` dictionary for cost of electricity, each key corresponds to a tier with value being \$/kWh timeseries
+- `energy_rate_tier_limits` dictionary for energy rate tier limits, each key corresponds to a tier with value being kWh limit
```

Comment:
@Bill-Becker @rathod-b from the description it's not clear how periods are captured in this output (can have different tier limits within different periods...)

Reply:
Under current setup, the energy rate series provides time of use energy costs for a multi-period electric tariff as a vector with energy rates varying over timestep. By default, the `energy_rate_tier_limits` is a large number.  In case the rate has multiple periods and tiers, the tier limits are reflected via columns in this output, with key for highest tier always being a large number (or final tier).

If tier limits differ across periods in a given month, the model only captures one tier limit as of now. The REopt model would have to be modified to capture tier limit variations within a month.

Reply:
The row values of the Tier columns will show the different tier limits if they vary by month, but we do not capture if two or more periods in a given month have different tier limits. In that case, it will only show the first tier limit of the first period. If multiple periods in a given month all have the same tier limits, that will be correct. If different, it will report it inaccurately. We are going to make an issue to fix this in the future.